rpi_backlight/__init__.py

```diff
@@ -2,7 +2,8 @@
 from contextlib import contextmanager
 from pathlib import Path
 from tempfile import gettempdir
-from typing import Any, Callable, Generator, Union, TYPE_CHECKING
+from typing import Any, Callable, Generator, Union, Optional, TYPE_CHECKING
+from enum import Enum
```

Comment:
Minor request, please put the enum import between ` from contextlib import contextmanager` and `from pathlib import Path`